### PR TITLE
Use eval_batch_size for AutoEncoder loss stats

### PR DESCRIPTION
**Pull request by efajardo-nv**
## Description
- Updated `AutoEncoder` to use `eval_batch_size` instead of `batch_size` (training batch size) for calculating loss stats in `get_anomaly_score_losses`.  Using `eval_batch_size` is more appropriate since it will not impact the final trained model and affect model convergence. The default `eval_batch_size` (1024) is larger than default `batch_size` (256) and results in a speedup in training postproc (#816)
- Fixed a couple debug statements that were throwing string formatting errors.

## Checklist
[x] I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
[x] New or existing tests cover these changes.
[x] The documentation is up to date with these changes.

### CODE REVIEW

**Review by hsin-c — APPROVED** [review]
The changes look good to me.

File: morpheus/models/dfencoder/autoencoder.py [hunk 1]
```diff
@@ -905,11 +905,11 @@ def _fit_centralized(self, df, epochs=1, val=None, run_validation=False, use_val
 
                     # Early stopping
                     current_net_loss = net_loss
-                    LOG.debug('The Current Net Loss:', current_net_loss)
+                    LOG.debug('The Current Net Loss: {}'.format(current_net_loss))
```
Comment (mdemoret-nv) [inline morpheus/models/dfencoder/autoencoder.py:908]: Use lazy formatting for logger messages:
```suggestion
                    LOG.debug('The Current Net Loss: %s', current_net_loss)
```

File: morpheus/models/dfencoder/autoencoder.py [hunk 2]
```diff
@@ -905,11 +905,11 @@ def _fit_centralized(self, df, epochs=1, val=None, run_validation=False, use_val
...
 
                     if current_net_loss > last_loss:
                         count_es += 1
-                        LOG.debug('Early stop count:', count_es)
+                        LOG.debug('Early stop count: {}'.format(count_es))
```
Comment (mdemoret-nv) [inline morpheus/models/dfencoder/autoencoder.py:912]: Use lazy formatting for logger messages:
```suggestion
                        LOG.debug('Early stop count: %s', count_es)
```

**Review by mdemoret-nv — APPROVED** [review]
LGTM. Does this close #816? If so, this PR should be linked to that issue.

File: morpheus/models/dfencoder/autoencoder.py (hunk 1 above)
Comment (efajardo-nv) [reply morpheus/models/dfencoder/autoencoder.py:908]: @mdemoret-nv waiting on customer feedback before closing #816.
